src/components/vm/hostdevs/hostDevCard.jsx

```diff
@@ -95,40 +95,40 @@ function getSource(hostDev, nodeDevices, hostdevId) {
 
         // If there are 2 usb devices without specified bus/device and same vendor/product,
         // it's impossible to identify which one is the one referred in VM's XML
-        cells.push(getOptionalValue(device, `${hostdevId}-device`, _("Device")));
-        cells.push(getOptionalValue(bus, `${hostdevId}-bus`, _("Bus")));
+        cells.push(getOptionalValue(device, `device-${hostdevId}`, _("Device")));
```

Comment:
Just in case you push again: "test: id's cannot change with a number" subject is confusing; ITYM "change → start"